packages/wrangler/bin/wrangler.js

```diff
@@ -25,13 +27,23 @@ Consider using a Node.js version manager such as https://volta.sh/ or https://gi
     // TODO:
     // - should we log a warning here?
     // - maybe we can generate a certificate that concatenates with ours?
-    // - is there a security concern/should we cleanup after we exit?
     //
     //  I do think it'll be rare that someone wants to add a cert AND
     //  use cloudflare WARP, but let's wait till the situation actually
     //  arises before we do anything about it
   } else {
-    pathToCACerts = join(__dirname, "../Cloudflare_CA.pem");
+    const osTempDir = os.tmpdir();
+    const certDir = path.join(osTempDir, "wrangler-cert");
+    const certPath = path.join(certDir, "Cloudflare_CA.pem");
+    // copy cert to the system temp dir if needed
+    if (!fs.existsSync(certPath)) {
+      fs.mkdirSync(certDir, { recursive: true });
+      fs.writeFileSync(
+        certPath,
+        fs.readFileSync(path.join(__dirname, "../Cloudflare_CA.pem"), "utf-8")
+      );
+    }
```

Comment:
There is a chance (small I know) that the `wrangler-cert` folder exists but the certificate file does not, right?

So perhaps split this into two ifs.. One that creates the folder if it doesn't exist and one that creates the file in the folder if it doesn't exist.

Reply:
mkdirSync will silently pass if the directory already exists, so it's fine

Reply:
Oh sorry, I thought you were checking `certDir` not `certPath`.